WIP - MTV-1554: Release Notes for MTV 2.7.2 (#564)
kubev2v/forklift-documentation: documentation/modules/rn-27-resolved-issues.adoc

{project-first} 2.7 has the following resolved issues:

[id="resolved-issues-2-7-1_{context}"]
== Resolved issues 2.7.1

> 

Author: ```suggestion
.Disk transfer tasks are not completed although the migration is successful

In earlier releases of {project-short}, when migrating vSphere or OVA virtual machines (VMs), the disk transfer phase was completed successfully, but the individual tasks were not. This happened because the logs from virt-v2v did not fully scan and therefore missed the last indication that the copy phase was done. link:https://issues.redhat.com/browse/MTV-1310[(MTV-1310)]
```

{project-first} 2.7 has the following resolved issues:

[id="resolved-issues-2-7-2_{context}"]
== Resolved issues 2.7.2

.VMNetworksNotMapped error occurs after creating a plan from the UI with the source provider set to OpenShift Virtualization

> In earlier releases of {project-short}, after creating a plan with an OpenShift Virtualization source provider, the Migration Plan failed with the error `The plan is not ready - VMNetworksNotMapped`. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1201[(MTV-1201)]

Author: ```suggestion
In earlier releases of {project-short}, after creating a plan with an {virt} source provider, the Migration Plan failed with the error `The plan is not ready - VMNetworksNotMapped`. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1201[(MTV-1201)]
```

.Migration Plan for {virt} to {virt} missing the source namespace causing VMNetworkNotMapped error

> In earlier releases of {project-short}, when creating a Migration Plan for an OCPv to OCPv migration using the Plan Creation Form, the network map generated was missing the source namespace, which caused a `VMNetworkNotMapped` error on the plan. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1297[(MTV-1297)]

Author: ```suggestion
In earlier releases of {project-short}, when creating a Migration Plan for an {virt} to {virt} migration using the Plan Creation Form, the network map generated was missing the source namespace, which caused a `VMNetworkNotMapped` error on the plan. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1297[(MTV-1297)]
```
